src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -20,21 +30,68 @@ test.describe("Buzz Page", () => {
 
   test.afterEach(async ({ page }) => {
     buzzPage = new BuzzPage(page);
-    await buzzPage.deleteTheNewestPost()
+    await buzzPage.deleteTheNewestPost(false)
   });
 
   test("Post should be shared", async ({ page }) => {
+    const randomTitle = generateRandomString()
     await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
-    await buzzPage.sharePost(filePath);
+    await buzzPage.sharePost(filePath, randomTitle);
     await page.reload();
-    await expect(buzzPage.photoImg.first()).toBeAttached();
+    await expect(buzzPage.bodyTextPost.first()).toHaveText(randomTitle);
   });
 
   test("Post should be edited", async ({ page }) => {
+    const randomTitle = generateRandomString()
```

Comment:
It's a good approach to use a string that is random. One hint tho - minor one. In my experience it's worth building a string that looks something like this: "TestE2E + <random_string> or <random_uuid>". If for example, you operate on a database that is not completely dedicated to tests.. such naming allows you to filter out test data easily and remove it by some automatic flows that run against the database.

Reply:
will improve it